docs/src/modules/components/AppSearch.js

```diff
@@ -233,13 +233,15 @@ export default function AppSearch() {
               flexDirection: 'column',
             },
             '& .DocSearch-StartScreenTitle': {
-              padding: theme.spacing(1),
-              color: theme.palette.grey[600],
-              fontWeight: 600,
+              padding: theme.spacing(0, 1),
+              paddingBottom: theme.spacing(1),
+              fontSize: theme.typography.pxToRem(13),
+              fontWeight: 500,
+              color: theme.palette.text.secondary,
```

Comment:
Hey Benny! I tried to customize the Start Screen items but there was some stuff I couldn't do myself. I'm trying to follow this design idea where we have each item being a link (with the chevron) and each category has an icon (I'm using the same icon used on the SideNav - _Popular Searches_ being an exception, but since they're all components, I'm using the icon used on the Component folder).

Here's the [Figma link for the mockup.](https://www.figma.com/file/4uv2dT18rXJPZBbrbpw61q/Marketing-site?node-id=4648%3A25729) Let me know if we can pull this off. If not, it's okay too. We can cut it for releasing the search redesign quicker :) 

<img width="400" alt="Screen Shot 2021-09-17 at 18 16 17" src="https://user-images.githubusercontent.com/67129314/133854771-0679fe60-1734-4f0f-a050-2e68844041a1.png">